azkaban-exec-server/src/main/java/azkaban/execapp/FlowRampManager.java

```diff
@@ -665,8 +665,10 @@ public void start() {
     private void pollExecution() {
       if (this.pollingCriteria.shouldPoll()) {
         if (this.pollingCriteria.satisfiesUnsavedDataAvailableCriteria()) {
+          LOGGER.info("Saving Ramp Setting to Database.");
```

Comment:
For uniformity, either use "Saving and Loading" or "Save and Load". I prefer the latter.

Reply:
good catch. thanks.